packages/server/test/course.integration.ts

```diff
@@ -1143,10 +1143,8 @@ describe('Course Integration', () => {
         .post(`/courses/enroll_by_invite_code/123`)
         .send({
           email: user.email,
-          first_name: user.firstName,
-          last_name: user.lastName,
-          password: 'random_password',
           selected_course: 1,
```

Comment:
And that we also tested to ensure that we have to call the enroll endpoint with our first name, last name, and password lmao